chain/chain/src/store/mod.rs

```diff
@@ -211,6 +211,7 @@ pub trait ChainStoreAccess {
         target_shard_layout: &ShardLayout,
         block_hash: CryptoHash,
         last_chunk_height_included: BlockHeight,
+        filter_receipts: bool,
```

Comment:
Can you add a comment explaining the circumstances where this is set to true or false?